proxy/http/HttpTransact.cc

```diff
@@ -8856,75 +8856,100 @@ HttpTransact::update_size_and_time_stats(State *s, ink_hrtime total_time, ink_hr
 
   // update milestones stats
   if (http_ua_begin_time_stat) {
-    HTTP_SUM_TRANS_STAT(http_ua_begin_time_stat, milestone_difference_msec(milestones.sm_start, milestones.ua_begin))
+    HTTP_SUM_TRANS_STAT(http_ua_begin_time_stat, milestone_difference_msec(milestones.ms_get(TransactionMilestones::SM_START),
+                                                                           milestones.ms_get(TransactionMilestones::UA_BEGIN)))
```

Comment:
Here, at least, `milestone_difference_msec()` should take the enum values and do the `ms_get` internally. That would look a lot cleaner.